it/controllers/HasEmployeeBeenOnStatutoryLeaveControllerISpec.scala

```diff
@@ -0,0 +1,364 @@
+package controllers
+
+import assets.{BaseITConstants, PageTitles}
+import models.{EmployeeStarted, UserAnswers}
+import pages._
+import play.api.http.Status._
+import play.api.libs.json.Json
+import utils.LocalDateHelpers._
+import utils._
+import views.ViewUtils.dateToString
+
+import java.time.LocalDate
+
+class HasEmployeeBeenOnStatutoryLeaveControllerISpec
+    extends IntegrationSpecBase with CreateRequestHelper with CustomMatchers with BaseITConstants with ITCoreTestData {
+
+  val dayEmploymentStarted = "the day their employment started"
+
+  "GET /been-on-statutory-leave" should {
+
+    "employee is type 3" when {
+
+      "the day before employee is first furloughed is before 5th April 2020" must {
+
+        val firstFurloughDate = LocalDate.parse("2020-04-05")
+        val furloughStartDate = LocalDate.parse("2021-03-01")
+        val boundaryStart     = dateToString(apr6th2019)
+        val boundaryEnd       = dateToString(firstFurloughDate.minusDays(1))
+
+        val userAnswers: UserAnswers = emptyUserAnswers
+          .set(FurloughStartDatePage, furloughStartDate)
+          .success
+          .value
+          .set(EmployeeStartedPage, EmployeeStarted.OnOrBefore1Feb2019)
+          .success
+          .value
+          .set(FirstFurloughDatePage, firstFurloughDate)
+          .success
+          .value
```

Comment:
Is that how the scalafmt ends up auto formatiing this?

Reply:
unfortunately yes. it would be good to refactor the formatter at some point